fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -6468,5 +6473,115 @@ public LogicalPlan visitKillAnalyzeJob(DorisParser.KillAnalyzeJobContext ctx) {
         long jobId = Long.parseLong(ctx.jobId.getText());
         return new KillAnalyzeJobCommand(jobId);
     }
+
+    /**
+     * PasswordOption
+     */
+    public PasswordOptions visitPasswordOption(DorisParser.PasswordOptionContext ctx) {
+        int historyPolicy = -2;
```

Comment:
```suggestion
        int historyPolicy = PasswordOptions.UNSET;
```
init all options to UNSET

Reply:
fixed